### ISSUE TITLE
About the performance of PCPA

### ISSUE THREAD

**Issue by OSU-Haolin** [ykotseruba/PedestrianActionBenchmark#3]
Hi!

I used local context, speed, box, and pose in PCPA and trained the PCPA model with epoch = 80, LR = 5e-7. But I cannot get ACC = 0.85 in JAAD test dataset as described in the paper (I only got acc = 0. 79). Is there any detail I missed in the training process?

Thanks,
Haolin

**Comment by ykotseruba** [comment]
Hi, sorry it took me a while to look into this. We corrected a sampling error after submitting the final print of the paper. Because the number of samples in JAAD decreased, the results dropped by 1-2% accuracy. There was no negative effect on PIE. Training parameters to use for JAAD_all are 120 epochs with 5e-6 learning rate. Speed should not be used for training in JAAD, as mentioned in the paper..

**Comment by jinxing94** [comment]
Hi, I used  120 epochs with 5e-6 learning rate, but only get the results. 
acc:0.80 auc:0.78 f1:0.57 precision:0.46 recall:0.73

The config yaml I use is

model_opts:
  model: PCPA
  obs_input_type: [local_context, pose, box]
  apply_class_weights: True
  normalize_boxes: True
  generator: True
net_opts:
  num_hidden_units: 256
  global_pooling: avg
  regularizer_val: 0.0001
  cell_type: gru
  backbone: c3d
exp_opts:
  datasets: [jaad_all]
  batch_size: [8]
  epochs: [120]
  lr: [5.0e-6]
